File: 백준/Bronze/1546. 평균/평균.cc
#include <iostream>
using namespace std;

int main() {
	int val;
	cin >> val;
	int num[1000];

	for (int i = 0;i < val;i++) {
		cin >> num[i];
	}

	int max = num[0];

	for (int i = 0;i < val - 1; i++) {
		if (max < num[i + 1])
			max = num[i + 1];
	}

	double num1[1000];

	for (int i = 0;i < val;i++) {
		num1[i] = (num[i] / (double)max * 100);
	}

	double sum = 0;

	for (int i = 0;i < val;i++) {
		sum += num1[i];
	}

	cout << sum / val;
}

File: 백준/Bronze/1546. 평균/평균.py
numOfSubjects = int(input())
gradeOfSubjects = list(map(int, input().split()))

maxGrade = max(gradeOfSubjects)
sum = 0 

for i in gradeOfSubjects:
    sum = sum + i/maxGrade*100

print(sum/numOfSubjects)
